src/AstGen.zig

```diff
@@ -10926,6 +10976,7 @@ const Scope = struct {
     /// The category of identifier. These tag names are user-visible in compile errors.
     const IdCat = enum {
         @"function parameter",
+        @"function prototype parameter",
```

Comment:
I don't think there is any benefit to separating this from `@"function parameter"`.

Reply:
> I don't think there is any benefit to separating this from `@"function parameter"`.

I think with such wording compiler diagnostics are more accurate. But I can delete it if necessary